api_types.ts

```diff
@@ -701,6 +702,11 @@ export interface Sync {
    * The dynamic URL is likely necessary for determining which API resources to fetch.
    */
   dynamicUrl?: string;
+
+  /**
+   * {@link MetadataContext}
+   */
+  metadataContext?: MetadataContext;
```

Comment:
Maybe a name like `parameters` would make it more clear what'll be here? cc @ekoleda-codaio

Reply:
Yeah... I'm curious if @alexd-codaio had a reason for not using an interface name more like `Parameters` when he first introduced MetadataContext?